test/functionalTest/cases/2933_PATCH_offending_attributes/PATCH_offending_attributes.test

```diff
@@ -0,0 +1,171 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+PATCH /v2/entities/ID/attrs fails must give offending attributes
+
+
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+
+--SHELL--
+
+#
+# 01. POST /v2/entities Entity creation
+# 02. 02. PATCH /v2/entities/Room1/attrs, to update existing attribute (temperature) and non existing attribute (pressur), resulting in 422 Unprocessable error
+# 03. PATCH /v2/entities/Room1/attrs, to update existing attribute (temperature) and non existing attributes (temper and pressur), resulting in422 Unprocessable error'
```

Comment:
Typos:

```suggestion
# 02. PATCH /v2/entities/Room1/attrs, to update existing attribute (temperature) and non existing attribute (pressur), resulting in 422 Unprocessable error
# 03. PATCH /v2/entities/Room1/attrs, to update existing attribute (temperature) and non existing attributes (temper and pressur), resulting in 422 Unprocessable error'
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3900/commits/b1a3dbb213a5fd7f4613782d341cb12b4e2086df